src/main/java/ua/danit/final_project/dto/ShiftCommentDto.java

```diff
@@ -0,0 +1,48 @@
+package ua.danit.final_project.dto;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import ua.danit.final_project.entities.Employee;
+import ua.danit.final_project.entities.Position;
+import ua.danit.final_project.entities.ShiftComment;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Getter
+@EqualsAndHashCode
+@ToString
+public class ShiftCommentDto implements Serializable {
+
+  private final Long id;
+
+  private final String text;
+
+  private final Timestamp date;
+
+  private final String forename;
+
+  private final String surname;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "surname" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=92&issues=AWWktx5ofVFgtsXasUWr&open=AWWktx5ofVFgtsXasUWr)

[ISSUE_KEY]: AWWktx5ofVFgtsXasUWr